### DISCUSSION TITLE
Buderus Logamatic + Logavent

### DISCUSSION THREAD

**Discussion by caveman2003** [emsesp/EMS-ESP32#861] [General]
Hi all, 

sorry i was not able to find an good translation for: "Kontrollierte Wohnraumlüftung (kwl)" so i used the brand name ...

Both devices heatpump and kwl are connected via EMS (EMS+ i hope...) and the question is ... can i read / write data to kwl? 
Heatpump is working perfectly in EMS-ESP, but from kwl (Logavent) no data ... 

i have seen a discussion about unused / ignored ems telegrams can one of those from Logavent? 

thanks for any information :-)

**Comment by proddy** [comment]
What type of device is it? Capture some logs with the missing device warnings and telegrams and we'll see what we can do.

**Reply by caveman2003** [reply]
Hi Proddy, 
after looking more in detail to the device and the connections behind i am not sure if it possible to control the device using EMS... 
the device itself (kwl) is Buderus Logavent HRV2-350 that have an RS-485 Modbus, an RJ-45 LAN (from product documentation not in use ...!?? need to open the device to see if this is correct) ), an antenna connection (for rc), an digital in (??). 
The RS-485 is connected to an Buderus Logamatic web KM200 HRV ... this is the device that make the connection to Buderus myVent app (in use, working).
Also found that this device is an OEM from Pluggit avent P Series or at least the control board is identical. 
There is also an good description of the modbus (unfortunatelay not working for me ... need to dig deeper)

so i think we can close the discussion, because not related to EMS ...
